### DISCUSSION TITLE
【原图】提瓦特大地图 填图

### DISCUSSION THREAD

**Discussion by Sqrootof** [share121/inter-knot#2278] [其他]
![提瓦特大地图4 8压缩版](https://github.com/user-attachments/assets/4ce7f93d-b40e-479e-84e8-f27591640c0e)
**应该是全网制作最好的图了。**<img src="https://img-static.mihoyo.com/communityweb/upload/dcbfa99981e274d48dacae36d0c1f249.png" alt="咕咕咕" width="75" >
更高清的版本请从百度网盘下载：
链接：https://pan.baidu.com/s/1PIuoupYPqqc4YRdYFWGzkg?pwd=twt1 
提取码：twt1

*帖子的图是10800\*8100，大小8.33mb，格式JPG质量5。
链接里有18000\*13500，大小16.54mb，格式JPG质量5，和18000\*13500，大小105.32mb，格式PNG的高质量图。*

**Comment by Hongxuli2006** [comment]
这么好的帖子为啥没流量？

**Reply by Hongxuli2006** [reply]
原来刚发

**Comment by Sqrootof** [comment]
顶

**Comment by Genshin-open** [comment]
不是哥们用百度网盘我提取一晚上都出不来吧

**Reply by Sqrootof** [reply]
github直传限制10mb大小，只能网盘。不过根据经验不开会员，100兆的图只用一二十分钟吧，建议下那个16兆的图，清晰度也足够，网盘下的也很快

**Reply by Myitian** [reply]
> github直传限制10mb大小，只能网盘。不过根据经验不开会员，100兆的图只用一二十分钟吧，建议下那个16兆的图，清晰度也足够，网盘下的也很快

哈哈，我 10 秒下完
我用 PS 稍微缩了一下挂 GitHub 上了

**Comment by Myitian** [comment]
## 原图（压缩分卷）
[提瓦特大地图4.8精品版.zip.001](https://raw.githubusercontent.com/Myitian/-01/main/提瓦特大地图4.8精品版/提瓦特大地图4.8精品版.zip.001)
[提瓦特大地图4.8精品版.zip.002](https://raw.githubusercontent.com/Myitian/-01/main/提瓦特大地图4.8精品版/提瓦特大地图4.8精品版.zip.002)

## 16000x12000 版（91.95 MB）
![提瓦特大地图4.8精品版.16kx12k](https://raw.githubusercontent.com/Myitian/-01/main/提瓦特大地图4.8精品版/提瓦特大地图4.8精品版.16kx12k.png)

**Reply by Sqrootof** [reply]
这招确实挺方便，一点击就下载了<img src="https://img-static.mihoyo.com/communityweb/upload/bf7e7cae6c42a727faa3d5e76b4a4c1c.png" alt="帕姆_比心" width="75" >

**Comment by Sqrootof** [comment]
再顶111<img src="https://img-static.mihoyo.com/communityweb/upload/5c433e340acca5043629fc84299a326a.png" alt="三月七_生气" width="75" >

**Comment by Sqrootof** [comment]
再顶333
